src/main/java/lotto/view/InputView.java

```diff
@@ -0,0 +1,19 @@
+package lotto.view;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class InputView {
+
+    private static final BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+
+    public static String getUserInput() {
+        try {
+            return br.readLine().trim();
+        } catch (IOException ie) {
+            getUserInput();
```

Comment:
```suggestion
            return getUserInput();
```

해당 코드는 의도대로 동작하지 않고 있을 것 같습니다!

Reply:
넵 맞습니다 ㅠ